platform/ui/src/components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { Icon, Typography } from '@ohif/ui';
+
+const Dropdown = ({ titleElement, title, list }) => {
+  const [open, setOpen] = useState(false);
+  const element = useRef(null);
+
+  const renderTitleElement = () => {
+    if (titleElement) {
+      return titleElement;
+    }
+
+    return (
+      <div className="flex">
+        <Typography>{title}</Typography>
+        <Icon name="chevron-down" className="text-white ml-1" />
+      </div>
+    );
+  };
+
+  const toggleList = () => {
+    setOpen(s => !s);
+  };
+
+  const handleClick = e => {
+    if (element.current && !element.current.contains(e.target)) {
+      setOpen(false);
+    }
+  };
+
+  const renderList = () => {
+    const itemsAmount = list.length;
+
+    return (
+      <div
+        className={classnames(
+          'absolute origin-top-right transition duration-300 transform top-100 right-0 mt-2 z-10 bg-primary-dark border border-secondary-main rounded shadow',
```

Comment:
To avoid a huge list of classes I'm breaking them into separate names variables in another project like:
```
const flex = 'flex flex-direction.... all stuff related to flex';
const border = 'border border-top. shadow.. all stuff related to border';
const font = 'font font-whatever etc...';
const theme = 'bg-color or something';
```
And then use it like: classNames(font, theme, whatever...)

Let me know what you think about this approach, I know it a lot of variables but I find it easier to reason about what style i have to update.

Reply:
This is an interesting approach. We're currently trying to group class names by subject. I think this is something we can definitely discuss but for the future, to avoid inconsistency by having different approaches for now.